docs/getting-started-guides/ubuntu.md

```diff
@@ -0,0 +1,177 @@
+# Kubernetes deployed on ubuntu nodes
+
+This document describes how to deploy kubernetes on ubuntu nodes, including 1 master node and 3 minion nodes, and people uses this approach can scale to **any number of minion nodes** by changing some settings with ease. Although there exists saltstack based ubuntu k8s installation ,  it may be tedious and hard for a guy that knows little about saltstack but want to build a really distributed k8s cluster. This new approach of kubernets deployment is much more easy and automatical than the previous one.
+
+[Cloud team from ZJU](https://github.com/ZJU-SEL) will keep updating this work.
+
+### **Prerequisites：**
+*1 The minion nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge* 
+
+*2 All machines can communicate with each orther, no need to connect Internet (should use private docker registry in this case)*
+
+*3 These guide is tested OK on Ubuntu 14.04 LTS 64bit server, but it should also work on most Ubuntu versions*
+
+*4 Dependences of this guide: etcd-2.0.0, flannel-0.2.0, k8s-0.15.0, but it may work with higher versions*
+
+*5 All the remote servers can be ssh logged in without a password by using key authentication* 
+
+
+### **Main Steps**
+#### I. Make *kubernetes* , *etcd* and *flanneld* binaries
+
+On your laptop, copy `cluster/ubuntu` directory to your workspace.
```

Comment:
we can use the get.k8s.io or in fact cluster/get-kube.sh to get the latest k8s, but we still need etcd and flannel binaries.

get.k8s.io is downloading k8s from https://storage.googleapis.com/kubernetes-release/release/${release}/kubernetes.tar.gz. we are downloading the k8s directly from github.They are the same I assume.Besides our way can customize the k8s version,while the get.k8s.io does not.